apps/web/src/components/utils/SuccessOrError.tsx

```diff
@@ -0,0 +1,19 @@
+import { showNotification } from '@mantine/notifications';
+import { Check } from '../../design-system/icons/general/Check';
+import { ErrorIcon } from '../../design-system/icons';
+
+export function show(state: 'success' | 'error', message: string) {
```

Comment:
```suggestion
export function successMessage(message: string) {
```

A small API suggestion, we can have two functions: 
- sucessMessage
- errorMessage

Instead of passing a param to a show function, wdyt? @ainouzgali

Reply:
Yes, I like this! @AlexVCS and it would also make it easier with all the complicated conditions about colorScheme and success/error

Reply:
Ok yeah that's good and I would like to make it less complicated. I'm not sure how to implement it in other parts of that function, but am figuring it out.